### PR TITLE
Improve WSIReader

### PR DESCRIPTION
**Pull request by drbeh**
Fixes #1695 

### Description
Update WSIReader to accept whole slide image level in `__init__`.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] In-line docstrings updated.

### CODE REVIEW

File: monai/data/image_reader.py [hunk 1]
```diff
@@ -717,7 +722,7 @@ def get_data(
         img,
         location: Tuple[int, int] = (0, 0),
         size: Optional[Tuple[int, int]] = None,
-        level: int = 0,
+        level: Optional[int] = None,
```
Comment (Nic-Ma) [inline monai/data/image_reader.py:725]: Could you please help add unit tests for different levels?

Thanks.

Comment (drbeh) [reply monai/data/image_reader.py:725]: There are already unit tests for levels 0 and 2. https://github.com/Project-MONAI/MONAI/blob/dev/tests/test_cuimage_reader.py

Comment (drbeh) [reply monai/data/image_reader.py:725]: @Nic-Ma, do you think that more is needed or they are fine?

Comment (Nic-Ma) [reply monai/data/image_reader.py:725]: OK, looks good to me.

Thanks.

**Review by Nic-Ma — APPROVED** [review]

**Comment by drbeh** [comment]
@wyli can you run `/build` here? thanks

**Pull request merged**